component/service/src/main/java/org/exoplatform/social/service/rest/PeopleRestService.java

```diff
@@ -299,8 +300,27 @@ public Response suggestUsernames(@Context UriInfo uriInfo,
 
         return Util.getResponse(nameList, uriInfo, mediaType, Response.Status.OK);
       }
+      // Search in member of space first

```

Comment:
Check what is the effect of next block 
// Includes spaces the current user is member.